documentation/API_documentation/QoD_API.md

```diff
@@ -0,0 +1,178 @@
+<br>
+
+#### QoD Delete QoS Session
+
+| **Deleting QoS session** |
+| ---------------------------- |
+| **HTTP Request**<br>  DELETE\<base-url>/qod-api/v0/sessions/{sessionId}<br>**Query Parameters**<br>  No query parameters are defined.<br>**Path Parameters**<br>  sessionId: Session ID that needs to be terminated.<br>**Request Body Parameters**<br>  No request body parameters are defined.<br><br>**Response**<br> **204:** Session deleted<br> **401:** Un-authorized. <br> **403:** Forbidden.<br> **404:** Session not found.<br> **503:** Service temporarily unavailable.|
+
+### 4.4 Errors
+
+Since CAMARA QoD API is based on REST design principles and blueprints, well defined HTTP status codes and families specified by community are followed [[4]](#4).
+
+Details of HTTP based error/exception codes for the QoD API are described in Section 4.3 of each API REST based method.
+Following table provides an overview of common error names, codes, and messages applicable to QoD API.
+
+| No | Error Name | Error Code | Error Message |
+| --- | ---------- | ---------- | ------------- |
+|1	|400 |	INVALID_INPUT |	"Expected property is missing: ueId.msisdn" |
+|2	|400 |	INVALID_INPUT |	"Expected property is missing: ueId.ipv4addr" |
+|3	|400 |	INVALID_INPUT |	"Expected property is missing: ueId.ipv4addr or ueId.ipv6addr" |
+|4	|400 |	INVALID_INPUT |	"Expected property is missing: uePorts" |
+|5	|400 |	INVALID_INPUT |	"Expected property is missing: qos" |
+|6	|400 |	INVALID_INPUT |	"Ranges not allowed: uePorts" |
+|7	|401 |	UNAUTHORIZED |	"No authorization to invoke operation" |
+|8	|403 |	FORBIDDEN |	"Operation not allowed" |
+|9	|404 |	NOT_FOUND |	"Session Id does not exist" |
+|10	|409 |	CONFLICT |	"Another session is created for the same UE" |
+|11	|500 |	INTERNAL |	"Session could not be created" |
```

Comment:
```suggestion
|11	|500 |	Internal Server Error |	"Session could not be created" |
```
1: The status code should be corrected. 
2: Does this error only occur, when the session cannot be created?

Reply:
It's just reflecting the current YAML v0.8.0 ... so need to be changed there first and can then be adapted in the documentation.